.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should we include this?

Reply:
Nope. Sorry should double check when i make a PR. No idea how my VScode configured for Python code